src/blocked.c

```diff
@@ -316,7 +337,10 @@ void serveClientsBlockedOnSortedSetKey(robj *o, readyList *rl) {
                          receiver->lastcmd->proc == bzpopminCommand)
                          ? ZSET_MIN : ZSET_MAX;
             unblockClient(receiver);
+            const ustime_t start = server.ustime;
             genericZpopCommand(receiver,&rl->key,1,where,1,NULL);
+            const ustime_t reply_us = ustime()-start;
```

Comment:
maybe it's a good chance to use the new Monotonic clock (see #7644).
and take this opportunity to use it in `call()` too.

Reply:
addressed in the latest commit.